### PR TITLE
[FIX] Replace React.DOM render with createRoot

### PR DESCRIPTION
**Pull request by luluiz**
**User-Facing Changes**

N/A

**Description**
It was removed from the deprecated React.DOM and replaced by createRoot.  
It removed StrictMode as it caused duplicate execution in the application's dialog.

**Checklist**

- [x] The web version was tested and it is running ok
- [x] The desktop version was tested and it is running ok

### CODE REVIEW

**Review by laisspportugal — CHANGES_REQUESTED** [review]
There is an error when running the web application when using this branch.
Can you please take a look on that?
image

**Comment by luluiz** [comment]
> There is an error when running the web application when using this branch. Can you please take a look on that? image

Since we started using the concurrent model applied by React's createRoot, I needed to adapt the Chart component's update behaviour to wait for the next cycle using queueMicrotask, which has better performance than setTimeout in this type of scenario.

File: packages/suite-base/src/panels/createSyncRoot.test.tsx [hunk 1]
```diff
@@ -5,51 +5,43 @@
...
 
-    const unmountCb = createSyncRoot(<TestComponent />, container);
-    expect(screen.queryAllByText(textTest)).toHaveLength(1);
-
-    unmountCb();
-    expect(screen.queryAllByText(textTest)).toHaveLength(0);
+    const text = "Unmount Component Test";
+    const TestComponent = () => <div>{text}</div>;
+    act(() => {
+      setTimeout(() => {
```
Comment (aneuwald-ctw) [inline packages/suite-base/src/panels/createSyncRoot.test.tsx:38]: Can queueMicrotask be used here as well to keep consistency?

Comment (luluiz) [reply packages/suite-base/src/panels/createSyncRoot.test.tsx]: Yes, sure.

**Review by aneuwald-ctw — APPROVED** [review]
LGTM. Liked the "queueMicrotask"

**Review by laisspportugal — APPROVED** [review]

**Pull request merged**